Project.toml

```diff
@@ -4,6 +4,7 @@ authors = ["rafaelorozco <rao9787@gmail.com>", "Mathias Louboutin <mathias.loubo
 version = "0.2.0"
 
 [deps]
+IterativeSolvers = "42fd0dbc-a981-5370-80f2-aaf504508153"
```

Comment:
Dependency should be minimal to the API, the examples's package should be separate

Reply:
Absolutely true my bad 
thank you for noticing